cloudflare/resource_cloudflare_access_service_tokens.go

```diff
@@ -19,6 +20,8 @@ func resourceCloudflareAccessServiceToken() *schema.Resource {
 			State: resourceCloudflareAccessServiceTokenImport,
 		},
 
+		CustomizeDiff: resourceCloudflareAccessServiceTokenCustomizeDiff,
```

Comment:
is there a reason we're opting for a custom function here over something like [`customdiff.ForceNewIf`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/helper/customdiff?utm_source=godoc#ForceNewIf)?

Reply:
Thanks for the suggestion - you're correct... I can make this work with `ForceNewIf` instead of the custom function to simplify abit. 

Initially I tried `ForceNewIfChange` as suggested in the linked issue but it required access to `schema.ResourceDiff` which it didn't provide.